### PR TITLE
reword some strings

### PR DESCRIPTION
**Pull request by r10s**
this pr rewords some strings and slightly shifts menu items.

see commit messages for details.

### CODE REVIEW

File: src/renderer/components/helpers/ChatMethods.ts [hunk 1]
```diff
@@ -217,7 +217,7 @@ export async function createChatByContactIdAndSelectIt(
   )
 
   if (!chatId) {
-    throw new Error(window.static_translate('create_chat_error_desktop'))
+    throw new Error('could not create chat')
```
Comment (Simon-Laux) [inline src/renderer/components/helpers/ChatMethods.ts:220]: This error is in one of 3 code paths shown to the user, But anyways should not happen and is not useful, so no need to translate it (`createChatByContactId` would normally throw a more useful error before this if is reached I think).

File: src/renderer/components/helpers/ChatMethods.ts [hunk 2]
```diff
@@ -217,7 +217,7 @@ export async function createChatByContactIdAndSelectIt(
...
   }
```
Comment (Simon-Laux) [inline src/renderer/components/helpers/ChatMethods.ts:221]: Anyways thinking about it this if statement is a remnant of using the cffi, it is useless as createChatByContactId does not return `0`/`null`, it's return type is `Result<u32>`, so we can remove the whole if statement here.

**Review by Simon-Laux — APPROVED** [review]
LGTM

File: src/renderer/components/helpers/ChatMethods.ts (hunk 2 above)
Comment (r10s) [reply src/renderer/components/helpers/ChatMethods.ts:221]: even  better!

Comment (Simon-Laux) [reply src/renderer/components/helpers/ChatMethods.ts:221]: made a pr for it https://github.com/deltachat/deltachat-desktop/pull/3475

**Pull request merged**